### PR TITLE
JENKINS-61285 Fix blank page on configure clouds if no cloud plugin installed

### PR DESCRIPTION
**Pull request by timja**
See [JENKINS-61285](https://issues.jenkins-ci.org/browse/JENKINS-61285).

This was broken when it was moved to it's own page and converted to groovy, the syntax that is used doesn't work and just ended up rendering nothing.

### Screenshots

<details>

<summary>Before</summary>

image

</details>

<details>

<summary>After</summary>

image

</details>

### Proposed changelog entries

* Entry 1: JENKINS-61285, Fix blank page on configure clouds page if no cloud plugin installed
* ...

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a JIRA issue should exist and be labeled as `lts-candidate`

### CODE REVIEW

**Comment by daniel-beck** [comment]
Could you provide before/after screenshots?

**Comment by timja** [comment]
<details>

<summary>Before</summary>

image

</details>

<details>

<summary>After</summary>

image


</details>

File: core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy [hunk 1]
```diff
@@ -40,12 +40,8 @@ l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName) {
...
             st.adjunct(includes: "lib.form.confirm")
         } else {
-            p {
-                _("There are no cloud implementations for dynamically allocated agents installed.")
-                a(href: rootURL + "/pluginManager/available") {
-                    _("Go to plugin manager.")
-                }
-            }
+            p(_("There are no cloud implementations for dynamically allocated agents installed. "))
+            a(href: rootURL + "/pluginManager/available", _("Go to plugin manager."))
```
Comment (oleg-nenashev) [inline core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy]: Would be nice if we could apply the "cloud" or "cloud-provider" label filter in URL. Could be a great symbiosis with @daniel-beck 's labeling PR

**Review by oleg-nenashev — APPROVED** [review]
Would be nice if we could apply the "cloud" or "cloud-provider" label filter in URL. Could be a great symbiosis with @daniel-beck 's labeling PR

File: core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy (hunk 1 above)
Comment (timja) [reply core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy]: Is there a way to do that currently? all the filtering seems to be done client side from what I can tell

Comment (daniel-beck) [reply core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy]: Nice idea, but way out of scope.

**Review by daniel-beck — APPROVED** [review]
Thanks!

**Review by markjacksonfishing — APPROVED** [review]

File: core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy (hunk 1 above)
Comment (daniel-beck) [reply core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy]: We _could_ link to https://jenkins.io/doc/developer/extensions/jenkins-core/#cloud but that's almost user-hostile. That would be easy enough to do here.

I did something similar in the Build Authorization Monitor due to lack of alternatives.

**Comment by oleg-nenashev** [comment]
I plan to merge it tomorrow if no negative feedback

File: core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy (hunk 1 above)
Comment (oleg-nenashev) [reply core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy]: Anyway, can be done in a follow-up

**Comment by oleg-nenashev** [comment]
Changelog Generator breaks on this PR...

```
Checking range from jenkins-2.223 to 1e5112a106146ee4fd26b3975aa0c406f691ba51
/jenkins-changelog-generator/bin/generate-jenkins-changelog:49:in `match': invalid byte sequence in US-ASCII (ArgumentError)
	from /jenkins-changelog-generator/bin/generate-jenkins-changelog:49:in `block in <main>'
	from /jenkins-changelog-generator/bin/generate-jenkins-changelog:48:in `each_line'
	from /jenkins-changelog-generator/bin/generate-jenkins-changelog:48:in `<main>'
```

Code:

```ruby
diff = `git log --pretty=oneline #{previous_version}..#{new_version}`

diff.each_line do |line|
	pr = /#([0-9]{4,5})[) ]/.match(line)
```

**Pull request merged**